src/mongo/client/wire_protocol_writer.cpp

```diff
@@ -28,8 +28,11 @@ WireProtocolWriter::WireProtocolWriter(DBClientBase* client) : _client(client) {
 void WireProtocolWriter::write(const StringData& ns,
                                const std::vector<WriteOperation*>& write_operations,
                                bool ordered,
+                               bool bypassDocumentValidation,
                                const WriteConcern* writeConcern,
                                WriteResult* writeResult) {
+    invariant(bypassDocumentValidation == false);
```

Comment:
By saying 'invariant' here, you are asserting that this code is unreachable by user action. Is that true? Otherwise should be a uassert.